.github/workflows/dockerhub-released-image.yml

```diff
@@ -1,4 +1,4 @@
-name: released image to dockerhub
+name: released image to DockerHub
 on:
   release:
     types:
```

Comment:
There's an extra colon in line 7 of this file.

Reply:
Done.